snowy-silence-43C/index.html

```diff
@@ -5,10 +5,10 @@
   <meta name="viewport" content="width=device-width">
   <title>Very epic vue.js</title>
   <script src="https://unpkg.com/vue"></script>
-  <style>.notEpic{background-color:#FF0000;} .virus{color:#FF00FF;} .notVirus{color:#00FF00;}</style>
+  <style>.notEpic{background-color:coral;} .virus{color:red;} .notVirus{color:green;}</style>
```

Comment:
Coral's a much better color, and using word-based colors makes sense. Nice work 👍